src/legacy/core_plugins/console/np_ready/public/application/components/console_menu.tsx

```diff
@@ -103,38 +104,23 @@ export class ConsoleMenu extends Component<Props, State> {
     this.props.autoIndent(event);
   };
 
-  getCopyToast = () => {
-    const copyToast = [
-      {
-        title: i18n.translate('console.copyRequestAsCurl', {
-          defaultMessage: 'Request copied as cURL',
-        }),
-        color: 'success',
-      },
-    ];
-
-    const actualToastId = this.state.toastId;
-    this.setState({ toastId: this.state.toastId + 1 });
-
-    return {
-      id: actualToastId,
-      ...copyToast[0],
-    };
-  };
-
   addCopyToast = () => {
-    const toast = this.getCopyToast();
+    const toast: Toast = {
+      id: this.state.toastId,
+      title: i18n.translate('console.consoleMenu.copyRequestAsCurlMessage', {
+        defaultMessage: 'Request copied as cURL',
+      }),
+      color: 'success',
+    };
 
     this.setState({
-      // @ts-ignore
+      toastId: this.state.toastId + 1,
```

Comment:
Hmm, looks like this is not going to give us the addition behaviour we want (it's just going to create an ever growing string of `1`s). I can see that the toast settings requires a string id though. I recommend keeping the state reference a number, and changing L109 to something like:
```ts
      id: String(this.state.toastId),
```

Reply:
@jloleysens : done!